public/sample-data/mesenvs-immuno-xsm.rnk

```diff
@@ -0,0 +1,27 @@
+gene	rank
+C1QTNF1	21.0109517861972
+MAGEL2	18.9648789038834
+GEM	14.5664464860213
+FBLN1	11.8216165367384
+NCKAP5L	8.9837479204489
+RNASE4	6.58350148660325
+DZIP1L	5.51569846956306
+ITM2C	4.5735070683876
+SALL2	3.23759727592196
+FAM171A2	2.27741830214219
+PAK4	1.55554249645681
+CNKSR3	0.600022794111153
+ZNF679	0
+ZNF100	-0.0833094057827782
+TOR1B	-1.1337844250007
+RNF7	-2.22770975202441
+NDUFV2	-4.73097513116836
+DDIT3	-6.20075956818443
+DDX60	-7.23165518082732
+GBP1	-8.77775627864623
+NUDT16P1	-9.36142904483035
+CALN1	-10.7079701444496
+ATPAF1	-12.7812820237903
+TAP2	-14.8247058519209
+CSAG3	-16.7307784891732
+PSMB9	-18.3590574351204
```

Comment:
@chrtannus, this is your XSM sample file with both positive and negative values.